packages/devtools-network-console/src/ui/ResponseViewer/index.tsx

```diff
@@ -20,7 +20,7 @@ import ResponseBody from './ResponseBody';
 import ContainerWithStatusBar from 'ui/generic/ContainerWithStatusBar';
 import { HideUnless } from 'ui/generic/HideIf';
 import { DesignSystemProvider } from '@microsoft/fast-jss-manager-react';
-import WebSocketView from './WebSocket';
+import ConnectedWebSocketViewer from './WebSocket';
```

Comment:
Nit: The code base generally doesn't use the `Connected`- prefix on imports or in the case of JSX (that a component is connected is an implementation detail).